app/mono_pkg.yaml

```diff
@@ -8,13 +8,27 @@ stages:
         - format
         - analyze: --fatal-infos  .
   - unit_test:
-    - test: -P presubmit --total-shards 7 --shard-index 0
-    - test: -P presubmit --total-shards 7 --shard-index 1
-    - test: -P presubmit --total-shards 7 --shard-index 2
-    - test: -P presubmit --total-shards 7 --shard-index 3
-    - test: -P presubmit --total-shards 7 --shard-index 4
-    - test: -P presubmit --total-shards 7 --shard-index 5
-    - test: -P presubmit --total-shards 7 --shard-index 6
+    - group:
+      - command: sudo apt-get install webp
+      - test: -P presubmit --total-shards 7 --shard-index 0
+    - group:
+      - command: sudo apt-get install webp
+      - test: -P presubmit --total-shards 7 --shard-index 1
+    - group:
+      - command: sudo apt-get install webp
+      - test: -P presubmit --total-shards 7 --shard-index 2
+    - group:
+      - command: sudo apt-get install webp
+      - test: -P presubmit --total-shards 7 --shard-index 3
+    - group:
+      - command: sudo apt-get install webp
+      - test: -P presubmit --total-shards 7 --shard-index 4
+    - group:
+      - command: sudo apt-get install webp
+      - test: -P presubmit --total-shards 7 --shard-index 5
+    - group:
+      - command: sudo apt-get install webp
+      - test: -P presubmit --total-shards 7 --shard-index 6
```

Comment:
@isoos any concerns about this?

It's a bit of a hack, but now we actually run tests with screenshots doing their thing..

Reply:
Looks acceptable, but I'd imagine that we should only have the `apt-get install` part only once.

Reply:
I have no idea how to do this.